behaviors/services/consensus-context.md

```diff
@@ -69,6 +69,9 @@ Currently, a single instance per virtual chain per node.
 * Get the block reference timestamp from the `TransactionsBlock` header.
   
 * Execute the ordered transactions set by calling `VirtualMachine.ProcessTransactionSet` creating receipts and state diff.
+  * Using the block reference timestamp.
```

Comment:
* Using the block reference timestamp - unnecessary, understood from the interface.
  * ~~Using the ~~ExecutionMode set to EXECUTION_PROPOSE.